### PR TITLE
Cleanup code

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: src/inertia.js [hunk 1]
```diff
@@ -53,47 +53,57 @@ export default {
...
       cancelToken: this.cancelToken.token,
       headers: {
-        'Accept': 'text/html, application/xhtml+xml',
+        Accept: 'text/html, application/xhtml+xml',
         'X-Requested-With': 'XMLHttpRequest',
         'X-Inertia': true,
-        ...this.version ? { 'X-Inertia-Version': this.version } : {},
+        ...(this.version ? { 'X-Inertia-Version': this.version } : {}),
       },
-    }).then(response => {
```
Comment (reinink) [inline src/inertia.js]: I did all these changes, other than the `.then()` indentation changes. Could you revert those? <3

Comment (MichaelDeBoey) [reply src/inertia.js]: Just wanted to make the code a bit more clear (in my opinion) that's all 🙂

Comment (reinink) [reply src/inertia.js]: Fair enough, but I think I prefer the previous formatting.

Comment (MichaelDeBoey) [reply src/inertia.js]: Reverted 🙂

Comment (reinink) [reply src/inertia.js]: Yeah I meant all of them, not just this one. :)

File: src/inertia.js [hunk 2]
```diff
@@ -53,47 +53,51 @@ export default {
...
-        this.setState(page, replace)
-        return this.setPage(page).then(() => {
-          this.setScroll(preserveScroll)
-          this.hideProgressBar()
-        })
-      } else {
-        this.hideProgressBar()
-      }
     })
+      .then(response => {
```
Comment (reinink) [inline src/inertia.js]: Here too

File: src/inertia.js [hunk 3]
```diff
@@ -53,47 +53,51 @@ export default {
...
+        if (this.isInertiaResponse(response)) {
+          return response.data
+        } else {
+          this.showModal(response.data)
+        }
+      })
+      .catch(error => {
```
Comment (reinink) [inline src/inertia.js]: And here

File: src/inertia.js [hunk 4]
```diff
@@ -53,47 +53,51 @@ export default {
...
+          return this.hardVisit(true, error.response.headers['x-inertia-location'])
+        } else if (this.isInertiaResponse(error.response)) {
+          return error.response.data
+        } else if (error.response) {
+          this.showModal(error.response.data)
+        } else {
+          return Promise.reject(error)
+        }
+      })
+      .then(page => {
```
Comment (reinink) [inline src/inertia.js]: And here

File: src/inertia.js (hunk 1 above)
Comment (MichaelDeBoey) [reply src/inertia.js]: Done 🙂

File: src/inertia.js (hunk 2 above)
Comment (MichaelDeBoey) [reply src/inertia.js]: Done 🙂

File: src/inertia.js (hunk 3 above)
Comment (MichaelDeBoey) [reply src/inertia.js]: Done 🙂

File: src/inertia.js (hunk 4 above)
Comment (MichaelDeBoey) [reply src/inertia.js]: Done 🙂

**Comment by reinink** [comment]
Thanks!

**Comment by MichaelDeBoey** [comment]
With pleasure! 🙂

**Pull request merged**